### DISCUSSION TITLE
How to clear console between every execution

### DISCUSSION THREAD

**Discussion by joey-ful** [codesandbox/codesandbox-client#5344] [Q&A]
Hi!
I have my clear console option **on** (It has been on the whole time since **I never changed the default setting**)
However, it doesn't seem to be working.
How could I have my console cleared automatically between every execution?

![image](https://user-images.githubusercontent.com/52592748/104092299-66679a00-52c6-11eb-9baa-6f04eab8012f.png)
![image](https://user-images.githubusercontent.com/52592748/104092291-5bad0500-52c6-11eb-88ab-c6b6057317fb.png)

Thanks

**Comment by SaraVieira** [comment]
Hey!

There seems to be a bug, it is supposed to be cleaned between executions :/

**Comment by SaraVieira** [comment]
Hey!

Can you send us a video of this happening or a sandbox where I can test this on? Just created a small sandbox to test this and it seems to clean on every execution

**Comment by smhutch** [comment]
@joey-ful are you still experiencing this issue?

**Reply by oxk4r01** [reply]
I'm experiencing this issue too. Very frustrating...

**Reply by SuperIbm** [reply]
Guys, what do you think when can we expect this issue to be solved?

**Reply by JamesACS** [reply]
Hey @SuperIbm - We haven't been able to replicate the issue again from our end - would you be able to send us a screen recording to support@codesandbox.io?

**Comment by 201flaviosilva** [comment]
Hey there.
I don't know if this discussion is still active, but to solve the problem I use:
`console.clear();` :)
